lib/screens/edit_persona/edit_category_fields.dart

```diff
@@ -415,9 +415,6 @@ class _EditCategoryFieldsState extends State<EditCategoryFields> {
   Widget inputField(BasicData basicData, {bool isCustomField = false}) {
     return Slidable(
       key: UniqueKey(),
-      actionPane: SlidableDrawerActionPane(),
```

Comment:
Do we not need `SlidableDrawerActionPane ` anymore ?

Reply:
Is there any alternate UI which has to be rendered ?

Reply:
@sachins-geekyants The package has changed its usage and there will be no alternate UI